### PR TITLE
refactor(virtctl/imageupload): use Printf from cobra library

### PR DESCRIPTION
**Pull request by codingben**
### What this PR does

Use Printf the default from cobra library
to print user-facing logs.

Requested by @0xFelix to cleanup imageupload
command.

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by codingben** [comment]
@0xFelix @oshoval Can you please review?

**Comment by oshoval** [comment]
@0xFelix 
why is it better ? allows to use a more robust logger?

**Comment by oshoval** [comment]
beside the question
/lgtm

File: pkg/virtctl/imageupload/imageupload_test.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/golang/mock/gomock"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"github.com/spf13/cobra"
```
Comment (oshoval) [inline pkg/virtctl/imageupload/imageupload_test.go:16]: please keep the space between the import groups
it differ between the kubevirt ones and the global ones

Comment (codingben) [reply pkg/virtctl/imageupload/imageupload_test.go]: Done. Can you please take a look?

File: pkg/virtctl/imageupload/imageupload_test.go [hunk 2]
```diff
@@ -31,6 +32,8 @@ import (
 	"kubevirt.io/kubevirt/pkg/virtctl/utils"
 	"kubevirt.io/kubevirt/tests/clientcmd"
 	"kubevirt.io/kubevirt/tests/libstorage"
+
+	"github.com/spf13/cobra"
```
Comment (0xFelix) [inline pkg/virtctl/imageupload/imageupload_test.go:36]: Please put this into the block starting at L17 and you could also remove the empty line in L23.

**Review by 0xFelix — COMMENTED** [review]
Failures look related.

@oshoval In theory it allows to capture output more easily in unit tests. If it should be tested.

File: pkg/virtctl/imageupload/imageupload_test.go (hunk 2 above)
Comment (codingben) [reply pkg/virtctl/imageupload/imageupload_test.go:36]: Why do you want to have it together with k8s deps? I know that this is the order:
```
go deps
ginkgo deps
k8s deps
kv deps
... any other deps (e.g. third-party - cobra dep)
```

**Review by 0xFelix — COMMENTED** [review]
/approve

**Pull request merged**